common/jenkins-slaves/nodejs10-angular/yarnrc

```diff
@@ -0,0 +1 @@
+registry "NEXUS_HOST/repository/npmjs//"
```

Comment:
the // at the end is required to make sure one of them stays when the value is used

Reply:
I just wanted to ask ;)

Reply:
Maybe add a comment why it is like that for future puzzled readers? :)